roles/image_template/README.md

```diff
@@ -49,6 +49,9 @@ Role Variables
 | glance_image_provider        | UNDEF (mandatory if qcow_url is not used)            | Name of the glance image provider.                    |
 | glance_image            | UNDEF (mandatory if qcow_url is not used)               | This parameter specifies the name of disk in glance provider to be imported as template. |
 | template_prerequisites_tasks | UNDEF | Works only with qcow image. Specify a path to Ansible tasks file, which should be executed on virtual machine before creating a template from it. Note that qcow image must contain guest agent which reports IP address. |
+| template_version_name      | "base version" | Version name of the template. Allows to create different versions of a template.  |
```

Comment:
What if someone runs this role right after upgrading without specifying the `template_version_name` wouldn't it create a new template or add a name to the existing one? 
I am worried that this could cause unexpected behaviour. 
If it creates a new template I would keep the variable `UNDEF`.

Reply:
> What if someone runs this role right after upgrading without specifying the `template_version_name` wouldn't it create a new template or add a name to the existing one?

None: if the template already exists, nothing will be done, and if it doesn't, a new one will be created with the default `template_version_name` which is `"base version"`.

>I am worried that this could cause unexpected behaviour. If it creates a new template I would keep the variable `UNDEF`.

All the templates created without specifiying the version name have `"base version"` as the default value. This is by default in the sense that the API will put that name if you don't specify it. So the behavior in case `template_version_name` is not specified should be equivalent to the current behavior.